### PR TITLE
Fail build scripts if any command fails

### PR DESCRIPTION
**Pull request by bartlettc22**
Builds were succeeding even if the app didn't compile - I believe this will fix that

### CODE REVIEW

File: scripts/build.sh
Comment (lwahlmeier) [inline scripts/build.sh]: another thing I have ended up having to add to some of our scripts is `set -o pipefail` because if a command fails but its pipeing to another command it wont be caught by set -e

**Review by lwahlmeier — APPROVED** [review]
lgtm, idk if we need the pipefail in the comment I added, leave it up to you.

File: scripts/build.sh
Comment (bartlettc22) [reply scripts/build.sh]: Good call - I added that as well

**Pull request merged**
